src/covid_model_seiir_pipeline/ode_fit/task.py

```diff
@@ -51,34 +47,23 @@ def run_ode_fit(draw_id: int, ode_version: str):
         gamma1=fit_specification.parameters.gamma1,
         gamma2=fit_specification.parameters.gamma2,
         solver_dt=fit_specification.parameters.solver_dt,
-        spline_options={
-            'spline_knots': fit_specification.parameters.knots,
-            'spline_degree': fit_specification.parameters.degree,
-            'prior_spline_convexity': None if not fit_specification.parameters.concavity else 'concave',
-            'prior_spline_monotonicity': None if not fit_specification.parameters.increasing else 'increasing',
-            'spline_knots_type': fit_specification.parameters.spline_knots_type,
-            'spline_r_linear': fit_specification.parameters.spline_r_linear,
-            'spline_l_linear': fit_specification.parameters.spline_l_linear,
-        },
         day_shift=fit_specification.parameters.day_shift,
-        spline_se_power=fit_specification.parameters.spline_se_power,
-        spline_space=fit_specification.parameters.spline_space,
+
     )
 
-    # ----------------------- BETA SPLINE + ODE -------------------------------- #
-    # Start a Model Runner with the processed inputs and fit the beta spline / ODE
-    mr = ModelRunner()
-    mr.fit_beta_ode(beta_fit_inputs)
+    # Convert infections into beta and fit compartments to data.
+    ode_model = model.ODEProcess(beta_fit_inputs)
```

Comment:
From the port of https://github.com/ihmeuw/covid-model-seiir/blob/develop/src/covid_model_seiir/ode_model/ode_process.py.  Don't use model runner anymore.  Use the `ODEProcess` interface directly.